_episodes/21-lessons.md

```diff
@@ -434,22 +341,111 @@ but as Scott pointed out in
 *[Seeing Like a State][amazon-slas]*,
 large organizations invariably prefer uniformity to productivity.
 
-> ## Validate Learning Objectives
+## Software and Data Carpentry Lessons
+
+It would be nice to say that the Software and Data Carpentry lessons were all 
+developed using perfect reverse instructional design.  While that's not necessarily 
+true, all of the lessons are constantly being revised and edited with certain 
+core objectives in mind.  
+
+### Software Carpentry
+
+The main aim of the Unix shell lesson is to familiarize people with a handful of basic concepts
+that crop up in many other areas of computing:
+
+*   the notions of a path and a home directory
+*   the use of history and tab completion to save time (and prevent mistakes)
+*   manipulating text using `head`, `tail`, `grep`, and related tools
+*   combining existing tools using pipes instead of writing new ones
+*   using loops to repeat operations
+
+The aims of the version control lesson are to teach people:
+
+*   how to keep track of their work,
+*   how to collaborate with other people online, and
+*   enough about privacy and licensing that they can begin to make sensible decisions about
+    what to put where and how to share it.
+
+The ostensible aim of the programming lessons are to show people
+how to build modular programs out of small functions
+that can be read, tested, and re-used.
+However,
+these concepts turn out to be hard to convey to people
+who are still learning the syntax of a programming language
+(forest and trees),
+so in practice the programming lessons focus primarily on
+the mechanics of doing common operations in those languages.
+
+### Data Carpentry
```

Comment:
I wold remove hyperlinks to lesson material from the Data Carpentry section below.